### PR TITLE
User routes

### PR DESCRIPTION
**Pull request by zikowang**
| Status  | Type  | Env Vars Change |
| :---: | :---: | :---: |
| Ready | Hotfix | No |
## Motivation and Context
Adjust api routs of the desktop app.

## How Has This Been Tested?
locally

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement (non-breaking change which cleans up / improves existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have read the **CONTRIBUTING** document.
- [x] I have considered the accessibility of my changes (i.e. did I add proper content descriptions to images, or run my changes with talkback enabled?)
- [x] I have documented my code if needed

### CODE REVIEW

**Review by luwol03 — CHANGES_REQUESTED** [review]

File: src/utils/api.js [hunk 1]
```diff
@@ -14,6 +14,51 @@ export function setTokenHeader(token) {
...
+export const login = (data) => api.post("/user/login", data);
+export const register = (data) => api.post("/user/register", data);
+
+// User
+export const getProfile = () => api.get("/user");
+export const deleteUser = (userId) => api.delete(`/user/${userId}`);
+
+// Language package
+export const createPackage = (data) => api.post("/languagePackage", data);
+export const getPackages = (data) => api.get("/languagePackage", data);
```
[the remark this answers is not in the document]
Comment (zikowang) [reply src/utils/api.js:27]: default is false?

File: src/utils/api.js [hunk 2]
```diff
@@ -14,6 +14,51 @@ export function setTokenHeader(token) {
...
+  );
+export const getGroupVocabulary = (groupId) =>
+  api.get(`/group/${groupId}/vocabulary`);
+export const modifyVocabulary = (vocabularyId, data) =>
+  api.put(`/vocabulary/${vocabularyId}`, data);
+export const deleteVocabulary = (vocabularyId) =>
+  api.delete(`/vocabulary/${vocabularyId}`);
+
+// Query Vocabulary
+export const getQueryVocabulary = (languagePackageId, staged = true) =>
```
[the remark this answers is not in the document]
Comment (zikowang) [reply src/utils/api.js:61]: what should be the default of `limit`?

File: src/utils/api.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (luwol03) [reply src/utils/api.js:27]: yes

File: src/utils/api.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (luwol03) [reply src/utils/api.js:61]: I think 100 is a good value for now.

[the remark this answers is not in the document]
Comment (noctera) [reply src/utils/api.js]: On server-side, when limit is not defined in request, the default value is 100

**Review by luwol03 — APPROVED** [review]

**Pull request merged**